libs/expo/betterangels/src/lib/screens/AddEditClient/RelevantContacts/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { Colors, Spacings } from '@monorepo/expo/shared/static';
+import { Accordion, TextButton } from '@monorepo/expo/shared/ui-components';
+import { RefObject, useMemo } from 'react';
+import { useFieldArray, useFormContext } from 'react-hook-form';
+import { ScrollView, View } from 'react-native';
+import {
+  CreateClientProfileInput,
+  RelationshipTypeEnum,
+  UpdateClientProfileInput,
+} from '../../../apollo';
+import Contact from './Contact';
+
+interface IRelevantContactsProps {
+  expanded: undefined | string | null;
+  setExpanded: (expanded: undefined | string | null) => void;
+  scrollRef: RefObject<ScrollView>;
+}
+
+export default function RelevantContacts(props: IRelevantContactsProps) {
+  const { scrollRef, expanded, setExpanded } = props;
+
+  const { control } = useFormContext<
+    UpdateClientProfileInput | CreateClientProfileInput
+  >();
+
+  const { fields, append, remove } = useFieldArray({
+    control,
+    name: 'contacts',
+  });
+
+  const isRelevant = expanded === 'Relevant Contacts';
```

Comment:
```suggestion
  const isRelevantContacts = expanded === 'Relevant Contacts';
```

let's update this everywhere. `isRelevant` is too ambiguous

Reply:
updated